sdk/remoterendering/Azure.MixedReality.RemoteRendering/src/Authentication/AuthenticationEndpoint.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
You should still be able to reference these in their original location. Since we'll be moving to a package reference, I suppose it's possible that taking a copy might be a better strategy, but we haven't really talked about it.

Reply:
Yes, since we'll be built in a different pipeline, and the STS code won't trigger my pipeline, it would be very easy for an STS change to break me. I thought that maybe it was better to break the link.

Reply:
(Also note that I'm switching to a package reference with a fixed version.)

Reply:
I think a copy is best for now. If we come up with a more robust strategy, I can change it subsequnetly.